src/js/formapp.js

```diff
@@ -24,14 +24,14 @@ import {
 
 import 'js/formapp/components';
 
-import PreloadRegion from 'js/regions/preload_region';
-
 import 'scss/formapp/comment.scss';
 import 'scss/formapp/form.scss';
 import 'scss/formapp/formio-overrides.scss';
 import 'scss/formapp/print.scss';
 import 'scss/formapp/pdf.scss';
 
+import 'js/regions/preload.scss';
+
```

Comment:
Correct spot in the file for this import? Wasn't 100% sure where to place it.

Reply:
fine for me